### DISCUSSION TITLE
Huawei - new method for older Kirins

### DISCUSSION THREAD

**Discussion by tomaszk8266** [zenfyrdev/bootloader-unlock-wall-of-shame#14] [General]
I found this: https://xdaforums.com/t/guide-how-to-downgrade-on-huawei-mate-20-lite-no-paying-or-bs-required.4660445/
Didn't test it, but it may be a free method to unlock kirin 710/f, 970, 980, and even 810.

**Comment by SkyKuker** [comment]
Seems good to me. I tried this approach on my Huawei Nova 3i (Android 8.1, with updates to 9.1) running on Kirin 710, by flashing the `board` (**warning: this will erase everything on the device!**) with the `Skip (OEM/NV) update` ticked and running `fastboot oem unlock`. It worked! `Fastboot` now shows
```
PHONE Unlocked
FRP Unlock
```
A quick reminder: **don't update with the built-in Huawei OTA updater!** (update.app on SD and then launch update) **It will relock the bootloader and replace any kernels/recoveries you've installed!** Flash updates in the same tool to avoid relocking.
